.gitignore

```diff
@@ -1,7 +1,6 @@
 .idea/
 node_modules/
 src/
-package-lock.json
```

Comment:
Don't commit `package-lock.json` only `yarn.lock`

Reply:
@wemeetagain @dapplion we use npm in all of our github workflows, and we instruct to use it in README as well. Do we really want to switch to yarn?

Also I have to use node 14 and freeze dependencies to get through CI so I have to commit `package-lock.json`

Reply:
@wemeetagain Can you take a look?

Reply:
@tuyennhv please stick to yarn.lock in this PR and open an issue about switching to package-lock.json. This PR can't be +14000 lines and we can discuss this latter

Reply:
@dapplion I just rebase to master, we don't need to commit `package-lock.json` anymore